draft-ietf-deleg.md

```diff
@@ -163,8 +163,8 @@ The DELEG record uses a new resource record type, whose contents are identical t
 * The SVCB IPv4hint and IPv6hint parameters keep their key values of 4 and 6, but the presentation format with DELEG MUST be Glue4 and Glue6.
 * Glue4 and Glue6 records when present MUST be used to connect to the delegated name server.
 * The target of any DELEG record MUST NOT be '.'
-* The target of a DELEG INCLUDE record MUST be outside of the delegated domain.
-* The target of a DELEG DIRECT record MUST be a domain below the delegated domain.
+* The target of a DELEG INCLUDE record MUST be a name that is not subordinate to the delegated domain and MUST NOT contain any IP addresses.
+* If the target of a DELEG DIRECT record is a name that is subordinate to the delegated domain, it MUST also contain one or more IP addresses.
```

Comment:
Shall this directly refer to `ip4` or `ip6` parameters?

```suggestion
* If the target of a DELEG DIRECT record is a name that is subordinate to the delegated domain, it MUST also contain one or more IP address parameters such as ip4 or ip6.
```

maybe this is also calling for what is an “IP address parameter” that this can be referred to. It is obvious today… but hey when ip8 will be around the corner, this may get out of sync.

Reply:
It seems to be referenced in DNS terminology RFC9499: https://www.rfc-editor.org/rfc/rfc9499#section-9-1.24

At the same time, the existing glue refers to in-domain
https://www.rfc-editor.org/rfc/rfc9499#section-7-2.38